core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java

```diff
@@ -233,9 +232,8 @@ public void put(InputStream body, com.github.dockerjava.core.MediaType mediaType
         execute(request).close();
     }
 
-    protected RequestBody toRequestBody(InputStream body, @Nullable String mediaType) {
+    protected RequestBody toRequestBody(InputStream body, String mediaType) {
```

Comment:
Maybe we should replace this with `org.jetbrains.annotations.Nullable`? At least this PR would get us to a consistent state, from which point we can consider whether it's worth keeping the jetbrains library in place!

Reply:
Sure: https://github.com/testcontainers/testcontainers-java/pull/920/commits/1f5b77c65224fc448dc7773f2686bbee87d6e10a